gammapy/background/make.py

```diff
@@ -0,0 +1,470 @@
+        Detector Y bin edges.
+    detx_edges : `~astropy.coordinates.Angle`
+        Detector X bin edges.
+    """
+
+    # define cube binning shape
+    n_ebins = 20
+    n_ybins = 60
+    n_xbins = 60
+    if n_obs < 100:       
+        minus_bins = int(n_obs/10) - 10
+        n_ebins += minus_bins
+        n_ybins += 4*minus_bins
+        n_xbins += 4*minus_bins
+    bg_cube_shape = (n_ebins, n_ybins, n_xbins)
+
+    # define cube edges
+    energy_min = Quantity(0.1, 'TeV') # TODO: should this be overwriten by the energy threshold??!!!!
+    # TODO: should E_min (= energy_edges[0]) be equal to E_THRES??!!!
+    #energy_min = energy_threshold
+    energy_max = Quantity(80, 'TeV')
+    dety_min = Angle(-0.07, 'radian').to('degree')
+    dety_max = Angle(0.07, 'radian').to('degree')
+    detx_min = Angle(-0.07, 'radian').to('degree')
+    detx_max = Angle(0.07, 'radian').to('degree')
+    # TODO: the bin edges (at least for X and Y) should depend on the
+    #       experiment/observatory.
+
+    # energy bins (logarithmic)
+    log_delta_energy = (np.log(energy_max.value)
+                        - np.log(energy_min.value))/bg_cube_shape[0]
+    energy_edges = np.exp(np.arange(bg_cube_shape[0] + 1)*log_delta_energy
+                          + np.log(energy_min.value))
+    energy_edges = Quantity(energy_edges, energy_min.unit)
+    # TODO: this function should be reviewed/re-written, when
+    # the following PR is completed:
+    # https://github.com/gammapy/gammapy/pull/290
+
+    # spatial bins (linear)
+    delta_y = (dety_max - dety_min)/bg_cube_shape[1]
+    dety_edges = np.arange(bg_cube_shape[1] + 1)*delta_y + dety_min
+    delta_x = (detx_max - detx_min)/bg_cube_shape[2]
+    detx_edges = np.arange(bg_cube_shape[2] + 1)*delta_x + detx_min
+
+    if DEBUG > 1:
+        energy_edges = Quantity([0.01, 0.1, 1., 10., 100.], 'TeV') # log binning
+        dety_edges = Angle(np.arange(-5., 6., 1.), 'degree') # stops at 5
+        detx_edges = Angle(np.arange(-5., 6., 1.), 'degree') # stops at 5
+    if DEBUG:
+        print("energy bin edges", energy_edges)
+        print("dety bin edges", dety_edges)
+        print("detx bin edges", detx_edges)
+
+    return energy_edges, dety_edges, detx_edges
+
+
+def fill_events(observation_table, fits_path, events_cube, livetime_cube, DEBUG):
+    """Fill events and compute corresponding livetime.
+
+    Get data files corresponding to the observation list, histogram
+    the events and the livetime and fill the corresponding cube
+    containers.
+
+    Parameters
+    ----------
+    observation_table : `~gammapy.obs.ObservationTable`
+        Observation list to use for the histogramming.
+    fits_path : str
+        Path to the data files.
+    events_cube : `~gammapy.background.CubeBackgroundModel`
+        Cube container for the events.
+    livetime_cube : `~gammapy.background.CubeBackgroundModel`
+        Cube container for the livetime.
+    DEBUG : int
```

Comment:
This is not a good way to do logging.

Please change to this way of generating log messages everywhere:
https://gammapy.readthedocs.org/en/latest/development/index.html?highlight=logging#generating-log-messages